README.md

```diff
@@ -68,16 +68,26 @@ To successfully run scripts that require client credentials, you will need to fo
 ### Tooling
 
 - **[Python Guidelines — Creative Commons Open Source][ccospyguide]**
-- [Black][black]: the uncompromising Python code formatter
+- [Black][black]: the uncompromising Python code formatter, by [psf][psf]. It
+  is opinionated, but provides a consistent style across the codebase. It
+  also provides a `--check` option to check if the code is formatted correctly.
 - [flake8][flake8]: a python tool that glues together pep8, pyflakes, mccabe,
-  and third-party plugins to check the style and quality of some python code.
-- [isort][isort]: A Python utility / library to sort imports.
+  and third-party plugins to check the style and quality of some python code, by [PyCQA][pycqa].
+- [isort][isort]: A Python utility / library to sort imports. It provides a
+  command line utility, Python library and plugins for various editors to
+  automatically sort all your imports.
 
 [ccospyguide]: https://opensource.creativecommons.org/contributing-code/python-guidelines/
 [black]: https://github.com/psf/black
 [flake8]: https://gitlab.com/pycqa/flake8
 [isort]: https://pycqa.github.io/isort/
 
+- Helper scripts:
+  - [`./dev/tools.sh`](dev/tools.sh): install and update development tools
```

Comment:
This doesn’t correctly explain what the shell script does. Good documentation should correctly explain the script and give an example of its usage. An example suggestion will be.

> The `./dev/tools.sh` script is a helper script that runs isort, black, and flake8 to automatically format and check the codebase for style issues. 
> To use the script, simply run the following command.
> ```
> ./dev/tools.sh
> ```
> The script can also accept command-line arguments to specify specific files or directories to check. For example:
> 
> ```
> ./dev/tools.sh path/to/my/file.py
> ```
>